lib/SaferpayJson/Request/Container/SchemeToken.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ticketpark\SaferpayJson\Request\Container;
+
+use JMS\Serializer\Annotation\SerializedName;
+use JMS\Serializer\Annotation\Type;
+
+final class SchemeToken
+{
+    /**
+     * @var string
+     * @SerializedName("Number")
+     * @Type("string")
+     */
+    private $number;
+
+    /**
+     * @var int
+     * @SerializedName("ExpMonth")
+     * @Type("integer")
+     */
+    private $expMonth;
+
+    /**
+     * @var int
+     * @SerializedName("ExpYear")
+     * @Type("integer")
+     */
+    private $expYear;
+
+    /**
+     * @var string
+     * @SerializedName("AuthValue")
+     * @Type("string")
+     */
+    private $authValue;
+
+    /**
+     * @var string|null
+     * @SerializedName("Eci")
+     */
+    private $eci;
+
+    public function __construct(string $number, int $expMonth, int $expYear, string $authValue)
```

Comment:
`$authValue` should be optional, if I see correctly, and therefore not part of the constructor.

![Bildschirmfoto 2024-06-04 um 08 38 57](https://github.com/Ticketpark/SaferpayJsonApi/assets/260361/8afea38e-9c36-48f1-a2ae-2aa10ae15741)

Reply:
It is mandatory in http://saferpay.github.io/jsonapi/1.23/index.html#Payment_v1_Transaction_Initialize , seems they changed it later, but I have not found anything in later release notes. Do you want me to make it optional?

![ScreenShot-000886](https://github.com/Ticketpark/SaferpayJsonApi/assets/47786813/ce6447e7-8766-4a19-b4a7-de6a92846a5d)

Reply:
Ah great, it's fine therefore for v1.23 👍